### PR TITLE
ms - updated link to this repository

### PR DESCRIPTION
**Pull request by MaximillianStewart**
Changed link destination in footer to link user to this repository instead of course projects repo

### CODE REVIEW

File: frontend/src/tests/components/Nav/Footer.test.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ describe("Footer tests", () => {
     
         const href = screen.getByTestId("github-href");
         // console.log(href.href);
-        expect(href).toHaveAttribute("href", "https://github.com/ucsb-cs156-f22/f22-5pm-happycows");
+        expect(href).toHaveAttribute("href", "https://github.com/ucsb-cs156-s23/proj-happycows-s23-6pm-3");
```
Comment (pconrad) [inline frontend/src/tests/components/Nav/Footer.test.js:18]:
```suggestion
        expect(href).toHaveAttribute("href", "https://github.com/ucsb-cs156/proj-happycows");
```

File: src/main/resources/application-development.properties [hunk 2]
```diff
@@ -6,7 +6,7 @@ spring.datasource.password=password
 spring.h2.console.settings.web-allow-others=true
 spring.h2.console.enabled=true
 app.showSwaggerUILink=true
-app.sourceRepo=${SOURCE_REPO:${$env.SOURCE_REPO:https://github.com/ucsb-cs156/proj-courses}}
+app.sourceRepo=${SOURCE_REPO:${$env.SOURCE_REPO:https://github.com/ucsb-cs156-s23/proj-happycows-s23-6pm-3}}
```
Comment (pconrad) [inline src/main/resources/application-development.properties:9]:
```suggestion
app.sourceRepo=${SOURCE_REPO:${$env.SOURCE_REPO:https://github.com/ucsb-cs156/proj-happycows}}
```

File: src/main/resources/application.properties [hunk 3]
```diff
@@ -20,7 +20,7 @@ management.endpoints.web.exposure.include=mappings
 springfox.documentation.swagger.v2.path=/api/docs
 spring.jpa.hibernate.ddl-auto=update
 app.admin.emails=${ADMIN_EMAILS:${env.ADMIN_EMAILS:phtcon@ucsb.edu}}
-app.sourceRepo=${SOURCE_REPO:${$env.SOURCE_REPO:https://github.com/ucsb-cs156/proj-courses}}
+app.sourceRepo=${SOURCE_REPO:${$env.SOURCE_REPO:https://github.com/ucsb-cs156-s23/proj-happycows-s23-6pm-3}}
```
Comment (pconrad) [inline src/main/resources/application.properties:23]:
```suggestion
app.sourceRepo=${SOURCE_REPO:${$env.SOURCE_REPO:https://github.com/ucsb-cs156/proj-happycows}}
```

File: src/test/java/edu/ucsb/cs156/happiercows/controllers/SystemInfoControllerTests.java [hunk 4]
```diff
@@ -51,7 +51,7 @@ public void systemInfo__admin_logged_in() throws Exception {
         .builder()
         .showSwaggerUILink(true)
         .springH2ConsoleEnabled(true)
-        .sourceRepo("https://github.com/ucsb-cs156/proj-courses")
+        .sourceRepo("https://github.com/ucsb-cs156-s23/proj-happycows-s23-6pm-3")
```
Comment (pconrad) [inline src/test/java/edu/ucsb/cs156/happiercows/controllers/SystemInfoControllerTests.java:54]:
```suggestion
        .sourceRepo("https://github.com/ucsb-cs156/proj-happycows")
```

**Review by pconrad — CHANGES_REQUESTED** [review]
So this is setting the url to the one for your specific team.  There is a change needed, since the default is `proj-courses`, which is definitely wrong.

But you shoudn't set the default to the specific url for your team.  Instead, set it to 

```
https://github.com/ucsb-cs156/proj-happycows
```

The right way to do this so that your your team deployments point to your team's repo is to just override the local .env file on localhost, and/or create a config variable on your dokku deployments:

```
dokku config:set proj-happycows SOURCE_REPO=https://github.com/ucsb-cs156-s23/proj-happycows-s23-6pm-3
dokku config:set proj-happycows-qa SOURCE_REPO=https://github.com/ucsb-cs156-s23/proj-happycows-s23-6pm-3
```

The whole reason the code is structured like this is so that we do NOT need to change this for every team.  This can't be merged into the main branch, becuase then the main branch can't be merged into the long-term main branch, where

**Comment by MrJacobEisner** [comment]
Feel free to reopen, Im closing this to help cleanup our pr queue

**Pull request closed without merging**